meta-cube/recipes-support/overc-utils/source/cube

```diff
@@ -54,7 +52,7 @@ case ${command} in
 	;;
     cmd)
 	shift
-	cube-cmd $@
+	cube-cmd "$@"
```

Comment:
This shouldn't be needed. we have a bug in cube-cmd, if it isn't taking all the parameters via $@